exercises/bowling/example.go

```diff
@@ -29,63 +37,62 @@ func NewGame() *Game {
 // An error is possible depending on pin value and previous rolls.
 func (g *Game) Roll(pins int) error {
 	// Validate pin count on roll.
-	if pins > 10 {
+	if pins > pinsPerFrame {
 		return ErrPinCountExceedsPinsOnTheLane
 	}
 	if pins < 0 {
 		return ErrNegativeRollIsInvalid
 	}
-	if g.completedFrames() == 10 {
-		// Already completed 10 frames.
+	if g.completedFrames() == framesPerGame {
 		return ErrCannotRollAfterGameOver
 	}
 	// Record the roll.
 	g.rolls[g.nRolls] = pins
 	g.nRolls++
-	if pins == 10 && g.completedFrames() < 9 {
-		// Frames 1 .. 9 can be strikes with no problems.
+	if pins == pinsPerFrame && g.completedFrames() < (framesPerGame-1) {
+		// Frames before last one can be strikes with no problems.
 		g.completeTheFrame()
 		return nil
 	}
-	if g.rollsThisFrame() == 2 {
-		// Have counted 2 rolls on this frame.
-		if g.rawFrameScore(g.rFrameStart) > 10 {
-			// Unless we have completed 9 frames, cannot count > 10 for 1st 2 rolls.
-			if g.completedFrames() != 9 {
+	if g.rollsThisFrame() == maxRollsPerFrame {
+		// Have counted normal max rolls on a frame.
+		if g.rawFrameScore(g.rFrameStart) > pinsPerFrame {
+			// Unless we have completed all but last frame, cannot count > pinsPerFrame.
+			if g.completedFrames() != (framesPerGame - 1) {
```

Comment:
Interesting how gofmt will insert the spaces on _some_ instances of parenthesis use. I will go without them.